tests/k8s/multi-node/Vagrantfile

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.require_version ">= 1.8.3"
+
+$build = <<SCRIPT
+cd /home/vagrant/go/src/github.com/cilium/cilium/
+docker build -t cilium:build_test /home/vagrant/go/src/github.com/cilium/cilium
```

Comment:
We already have a way to build an image and push it to a registry, can we use this instead:

`tests/k8s/Vagrantfile:`
```
$build = <<SCRIPT
docker run -d -p 5000:5000 registry
cd /home/vagrant/go/src/github.com/cilium/cilium/
docker build -t cilium:build_test /home/vagrant/go/src/github.com/cilium/cilium
export ID=$(docker images | grep "build_test" | awk '{print $3}')
docker tag $ID localhost:5000/cilium:build_test
docker push localhost:5000/cilium:build_test
SCRIPT
```